prime-router/src/main/kotlin/Element.kt

```diff
@@ -268,7 +271,18 @@ data class Element(
                     // TODO Revisit: there may be times that normalizedValue is not an altValue
                     altDisplayToken ->
                         toAltDisplay(normalizedValue)
-                            ?: error("Schema Error: '$normalizedValue' is not in altValues set for $fieldMapping")
+                            ?: "".also {
+                                // Warn in the logs rather than actually throwing an exception.
+                                // This occurs in the Batch step, so no way to error back to the sender.
+                                println(
+                                    "Exception: Outgoing receiver schema ERROR:" +
+                                        " '$normalizedValue' is not in altValues set for $fieldMapping." +
+                                        " Replacing with empty-string in generated data for element $name" +
+                                        " and continuing to process." +
+                                        " Consider fixing by adding $normalizedValue to the " +
+                                        " alt valueset in schema ${schema?.name ?: ""}"
+                                )
+                            }
```

Comment:
Above is the main change in this PR:   replace the error() exception with logging.